tools/.travis-template.yml

```diff
@@ -77,70 +78,43 @@ script:
     (cd ${PYDIR}; cardboardlinter --refspec $TRAVIS_BRANCH -f static);
   fi
 
-# Conda-based tests
-# ~~~~~~~~~~~~~~~~~
-
-# This is needed to simulate the end-user situation, and it also creates packages for a
-# conda release.
-
-# Build the conda packages
-- conda build -q tools/conda.recipe
-- (cd ${PYDIR}; ./setup.py; conda build -q tools/conda.recipe)
-# Install Conda packages
-- conda install --use-local ${CONDA_PKG_NAME_CPP} ${CONDA_PKG_NAME_PY}
-# Run the unittests, using the installed package
-- ${CONDA_PREFIX}/libexec/${PROJECT_NAME}/test_${PROJECT_NAME}
-- (cd; nosetests ${PROJECT_NAME} -v --detailed-errors)
-
-# In-place building and testing
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-# This is allows for coverage analysis and dynamic linting. The compiler settings used
-# here are not suitable for releases, so we need to recompile and rerun the tests.
-
-# Install GCC compilers for in-place builds, even on OSX because clang does not manage to
-# compiler our C++ code.
-- conda install gcc libgcc;
-
-# PY project
-# Uninstall conda package, to be sure. The conda cpp package is still used.
-- conda uninstall ${CONDA_PKG_NAME_PY}
-- (cd ${PYDIR}; python setup.py build_ext -i --define CYTHON_TRACE_NOGIL)
-# Run nosetests without coverage.xml output. That file is broken by nosetests (pyx files
-# not include) and gets priority over .coverage, which contains everything.
-- (cd ${PYDIR}; nosetests ${PROJECT_NAME} -v --detailed-errors --with-coverage --cover-package=${PROJECT_NAME} --cover-tests --cover-erase --cover-inclusive --cover-branches; coverage xml -i)
-
-- if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-    (cd ${PYDIR}; cardboardlinter --refspec $TRAVIS_BRANCH -f 'dynamic');
+- if [[ $TRAVIS_TAG ]]; then
+    conda build -q tools/conda.recipe;
+    (cd ${PYDIR}; conda build -q tools/conda.recipe);
+    conda install --use-local ${CONDA_PKG_NAME_CPP} ${CONDA_PKG_NAME_PY};
+    ${CONDA_PREFIX}/libexec/${PROJECT_NAME}/test_${PROJECT_NAME};
+    (cd; nosetests ${PROJECT_NAME} -v --detailed-errors);
+    conda uninstall ${CONDA_PKG_NAME_PY};
+    conda uninstall ${CONDA_PKG_NAME_CPP};
+  else
+    ${DEPENDENCIES};
+    (mkdir debug; cd debug; cmake cmake -DCMAKE_BUILD_TYPE=debug ..; make);
```

Comment:
typo? cmake cmake

Reply:
It's always been there. I'll take it out though.